include/FastJetTopTagger.h

```diff
@@ -17,15 +17,37 @@
 #include <IMPL/LCCollectionVec.h>
 #include <EVENT/ReconstructedParticle.h>
 #include "IMPL/LCCollectionVec.h"
+#include <IMPL/ReconstructedParticleImpl.h>
```

Comment:
Please sort the includes. `LCCollectionVec.h` is already there, twice. (please remove the one with quotes)

Reply:
Done